README.md

```diff
@@ -24,9 +25,14 @@ This project contains:
 - [Contributing](#Contributing)
 - [Feedback](#Feedback)
 --- 
+## Client IDEs
+
+Eclipse LSP4Jakarta is consumed by the following IDEs:
+-  Eclipse IDE with [Liberty Tools for Eclipse](https://github.com/OpenLiberty/liberty-tools-eclipse#welcome-to-the-liberty-tools-for-eclipse-project)
```

Comment:
should we mention that it is the early release?

Reply:
I don't think we need to call out here that Liberty Tools for Eclipse is an early release, it is at the top of their readme